libbeat/cmd/instance/beat.go

```diff
@@ -474,6 +477,14 @@ func (b *Beat) Setup(bt beat.Creator, template, dashboards, machineLearning, pip
 			fmt.Println("Loaded Ingest pipelines")
 		}
 
+		if policy {
+			err := b.loadILMPolicy()
+			if err != nil {
+				return err
+			}
+			fmt.Println("ILM policy loaded for Beat")
```

Comment:
Nitpick - could you align with the other loading outputs to print `Loaded Index Lifecycle Management (ILM) policies`.

Reply:
done